addons/message-filters/userscript.js

```diff
@@ -0,0 +1,77 @@
+export default async function({addon,global,console}) {
+  let list = await addon.tab.waitForElement(".messages-social-list", {
```

Comment:
Does this work if you load more messages?

Reply:
Yes, that's why I need the setInterval.

Reply:
Instead, use an `await addon.tab.waitForElement` for new messages elements.

Reply:
Use `while(true)`

Reply:
I did. Thanks.